src/views/News/index.jsx

```diff
@@ -290,6 +297,19 @@ const StudentNews = () => {
     loadNews();
   }
 
+  async function handleNewsApprovalStatus(snid, newsStatusAccepted) {
+    // update the news item accepted status and give feedback
+    let result = await newsService.updateAcceptedStatus(snid, newsStatusAccepted);
+    let newStatus = newsStatusAccepted ? 'approve' : 'unapprove';
+    if (result === undefined) {
+      createSnackbar(`News Posting Failed to ${newStatus}`, 'error');
+    } else {
+      createSnackbar(`News Posting ${newStatus}d Successfully`, 'success');
```

Comment:
```suggestion
    let statusAction= newsStatusAccepted ? 'approve' : 'unapprove';
    if (result === undefined) {
      createSnackbar(`News Posting Failed to ${statusAction}`, 'error');
    } else {
      createSnackbar(`News Posting ${statusAction}d Successfully`, 'success');
```
The values here are technically not statuses, but actions to change the status. This rename would clarify that fact.